### PR TITLE
Rename Ansible Tower to Automation Controller

### PR DESCRIPTION
**Pull request by Lennonka**
_Ansible Tower_ was renamed to _Ansible Automation Controller_, which is a component of AAP.

**Note:** _Ansible Tower_ is also mentioned in a name of a user role and parameters, which is not changed in the code yet. This PR reflects that.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; planned orcharhino 6.4 on EL8 only)
* [x] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [x] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)

### CODE REVIEW

File: guides/common/attributes-satellite.adoc [hunk 1]
```diff
@@ -35,8 +35,8 @@
 :ansible-namespace-example: redhat.satellite._module_name_
 :ansible-namespace: redhat.satellite
 :ansiblefilepath: /usr/share/ansible/collections/ansible_collections/redhat/satellite/plugins/modules/
-:awx: Ansible Tower
-:awx-context: Ansible_Tower
+:awx: Ansible Automation Controller
+:awx-context: Ansible_Automation_Controller
```
Comment (maximiliankolb) [inline guides/common/attributes-satellite.adoc:39]: If we use dashes here, we could use this attribute in host names.

**Review by maximiliankolb — APPROVED** [review]
One tiny idea; rest LGTM

File: guides/common/attributes-satellite.adoc (hunk 1 above)
Comment (Lennonka) [reply guides/common/attributes-satellite.adoc:39]: I think it's unnecessarily long for host name examples.

**Comment by Lennonka** [comment]
Cherry-picked:
- 275f2a463..737fa6f7d  3.7 -> 3.7
- 2a96bc934..d6d4b8cef  3.6 -> 3.6
- 68e433393..ca4aa7bb0  3.5 -> 3.5
- 779e0e61a..fdcd02869  3.4 -> 3.4
- a99ae664d..952f006e1  3.3 -> 3.3
- e1f7b7dab..af5d51ad9  3.2 -> 3.2
- 66e545c9e..5150ebb55  3.1 -> 3.1

**Pull request merged**